client/my-sites/stats/wordads/index.jsx

```diff
@@ -161,39 +181,86 @@ class WordAds extends Component {
 							siteId={ siteId }
 							slug={ slug }
 						/>
-						<div id="my-stats-content" className="wordads">
-							<WordAdsChartTabs
-								activeTab={ getActiveTab( this.props.chartTab ) }
-								activeLegend={ this.state.activeLegend }
-								availableLegend={ this.getAvailableLegend() }
-								onChangeLegend={ this.onChangeLegend }
-								barClick={ this.barClick }
-								switchTab={ this.switchChart }
-								charts={ CHARTS }
-								queryDate={ queryDate }
-								period={ this.props.period }
-								chartTab={ this.props.chartTab }
-							/>
-							<StickyPanel className="stats__sticky-navigation">
-								<StatsPeriodNavigation
-									date={ queryDate }
-									hidePreviousArrow={
-										( 'day' === period || 'week' === period ) &&
-										moment( queryDate ).isSameOrBefore( '2018-10-01' )
-									} // @TODO is there a more elegant way to do this? Similar to in_array() for php?
-									hideNextArrow={ yesterday === queryDate }
-									period={ period }
-									url={ `/stats/ads/${ period }/${ slug }` }
-								>
-									<DatePicker
-										period={ period }
-										date={ queryDate }
-										query={ query }
-										statsType="statsAds"
-										showQueryDate
+
+						<div id="my-stats-content" className={ wrapperClasses }>
+							{ isNewFeatured ? (
+								<>
+									<StatsPeriodHeader>
+										<StatsPeriodNavigation
+											date={ queryDate }
+											hidePreviousArrow={
+												( 'day' === period || 'week' === period ) &&
+												moment( queryDate ).isSameOrBefore( '2018-10-01' )
+											} // @TODO is there a more elegant way to do this? Similar to in_array() for php?
```

Comment:
Did you mean something like `[ 'day', 'week' ].includes( period )`?

Reply:
> vertical spacing created by stats-date-picker__refresh-status.

Yes, I also noticed it and was trying to figure out the missing props `queryDate`.

> Did you mean something like [ 'day', 'week' ].includes( period )?

I think so, but it seems to be left by someone who worked on it before me. I was not trying to address it at the time.

I will tackle these fixes in a follow-up PR.

Reply:
Created an issue for it: https://github.com/Automattic/wp-calypso/issues/69781.